ONSdigital/ras-rm-docker-dev
# Add rm-reporting to docker

# Motivation and Context
rm-reporting wasn't part of the build.  This causes certain features in response-operations-ui to not work as the underlying service isn't there.

# What has changed




# How to test?
  - Start all the services using the updated compose files.  
  - Run the acceptance tests.  
  - Log into response-operations-ui
  - Go to Bricks survey
  - Click on the response chasing report.  It should download instead of giving a 500 error.


# Links
https://trello.com/c/tQE4Aziu/985-spike-add-rm-reporting-to-ras-rm-docker-dev-1-day
